sources/dracut/85sys-upgrade-redhat/do-upgrade.sh

```diff
@@ -48,7 +51,7 @@ do_upgrade() {
         # run leapp to proceed phases after the upgrade with Python3
         #PY_LEAPP_PATH=/usr/lib/python2.7/site-packages/leapp/
         #$NEWROOT/bin/systemd-nspawn $nspawn_opts -D $NEWROOT -E PYTHONPATH="${PYTHONPATH}:${PY_LEAPP_PATH}" /usr/bin/python3 $LEAPPBIN upgrade --resume $args
-        $NEWROOT/bin/systemd-nspawn $nspawn_opts -D $NEWROOT /usr/bin/python3 $LEAPP3_BIN upgrade --resume $args
+        $NEWROOT/bin/systemd-nspawn $nspawn_opts -D $NEWROOT /usr/bin/bash -c "mount -a; /usr/bin/python3 $LEAPP3_BIN upgrade --resume $args"
```

Comment:
Is it safe to chang the mount and pytho3 command like that? I.e. what will happend if `mount -a` fails?

Reply:
see the commit message. if it fails we are done as without mounted partitions the upgrade will fail in majority cases as well

Reply:
Duh omg right this is the problem I totally forgot that part